kiliautoml/utils/ultralytics/train.py

```diff
@@ -187,10 +185,6 @@ def ultralytics_train_yolov5(
                 data_path=data_path,
                 class_names=class_names,
                 number_classes=len(class_names),
-                kili_api_key=api_key,
```

Comment:
Also remove the unused arguments in `ultralytics_train_yolov5`, and in the calling hierarchy.

Reply:
nevermind, I thought `yaml_preparation` was doing something else.